### PR TITLE
Lambda with plain old Java 17 on GraalVM

### PR DESCRIPTION
**Pull request by mikrethor**

### CODE REVIEW

**Review by maxday — COMMENTED** [review]
Woot woot! Thanks a lot for the contribution 💯 
I just left some minor comments

File: s3-uploader/app.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ const { S3Client, PutObjectCommand } = require("@aws-sdk/client-s3");
         'dotnetcore31',
         'dotnet6',
         'go1x',
+        'graalvm_java17_on_provided_al2',
```
[the remark this answers is not in the document]
Comment (maxday) [reply s3-uploader/app.js:17]: I know this is duplication, I definitely need to refactor to make changes at one single place :) creating an issue for that right now

File: s3-uploader/runtimes/graalvm_java17_on_provided_al2/src/test/java/com/xavierbouclet/HandlerTest.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package com.xavierbouclet;
```
[the remark this answers is not in the document]
Comment (mikrethor) [reply s3-uploader/runtimes/graalvm_java17_on_provided_al2/src/test/java/com/xavierbouclet/HandlerTest.java]: done

File: s3-uploader/runtimes/graalvm_java17_on_provided_al2/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+RUN strip /tmp/graalvm-java17
+
+# zip the extension
+FROM ubuntu:latest as compresser
+RUN apt-get update
+RUN apt-get install -y zip
+RUN mkdir /package
+WORKDIR /package
+COPY --from=stripper /tmp/graalvm-java17 /package/graalvm-java17
+RUN zip -j code.zip /package/graalvm-java17
```
[the remark this answers is not in the document]
Comment (mikrethor) [reply s3-uploader/runtimes/graalvm_java17_on_provided_al2/Dockerfile]: done

**Review by maxday — APPROVED** [review]
Thanks a lot!

**Comment by anthonydahanne** [comment]
howdy!
I believe there's a more recent (`22.3.1 > 21.2.0`) GraalVM version around (see https://www.graalvm.org/release-notes/)

**Comment by maxday** [comment]
Thanks!
I think it's time for `dependabot` :)

**Pull request merged**